lib/registration.pm

```diff
@@ -867,6 +867,11 @@ sub scc_deregistration {
         quit_packagekit;
         wait_for_purge_kernels;
         assert_script_run('SUSEConnect --version');
+        if (get_var('UPGRADE_TARGET_VERSION') == '15-SP3') {
+            # https://bugzilla.suse.com/show_bug.cgi?id=1189543
```

Comment:
I don't think comments bug here is a must(code already have it), while it is better to comments why and how we workaround the bug.